### ISSUE TITLE
dulwich command line tool no longer works

### ISSUE THREAD

**Issue by inducer** [dulwich/dulwich#784]
```
Traceback (most recent call last):
  File "/home/andreas/.cache/pypoetry/virtualenvs/relate-courseware-on7htJeR-py3.8/bin/dulwich", line 8, in <module>
    sys.exit(main())
  File "/home/andreas/.cache/pypoetry/virtualenvs/relate-courseware-on7htJeR-py3.8/lib/python3.8/site-packages/dulwich/cli.py", line 713, in main
    if len(argv) < 1:
TypeError: object of type 'NoneType' has no len()
```
(`argv` would be passed as the first argument to `main`, but it defaults to `None`, which line 713 isn't prepared for)

**Comment by inducer** [comment]
dulwich 0.20.3, I forgot to add.

**Comment by jelmer** [comment]
On Sun, Jun 21, 2020 at 04:31:37PM -0700, Andreas Klöckner wrote:
> dulwich 0.20.3, I forgot to add.
Thanks for the report; this is fixed in master, I'll release 0.20.4
soon.

Jelmer

-- 
Jelmer Vernooĳ <jelmer@jelmer.uk>
PGP Key: https://www.jelmer.uk/D729A457.asc
